test/serializable_resource_test.rb

```diff
@@ -2,8 +2,13 @@
 
 module ActiveModel
   class SerializableResourceTest < Minitest::Test
+    class Profile < ::Model
```

Comment:
sister gems? there's a gem that uses `Model`? how aweful.  well, how about `SerializableModel` and `SerializableRecord` as the base classes